week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,17 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var conversionUSD = price * 1.4;
+  return addComission(conversionUSD);
+}
+function convertToBRL(price) {
+  var conversionBRL = price * 4.9;
+  return addComission(conversionBRL);
+}
+function addComission(conversion) {
```

Comment:
In awe at how clear and easy to understand your code is Madiha! Did you get the .toFixed to run fully?

Reply:
rhanks